src/main/fs.ts

```diff
@@ -1,5 +1,8 @@
-import { mkdir, mkdirSync, readdir, readFile, rename, stat, Stats, statSync, unlink } from 'fs';
-import { dirname, resolve } from 'path';
+import { app } from 'electron';
+import { mkdir, mkdirSync, readdir, readFile, rename, stat, Stats, statSync, unlink, writeFile } from 'fs';
+import { dirname, join, resolve } from 'path';
+
+export const sentryCachePath = join(app.getPath('userData'), 'sentry');
```

Comment:
Users can modify `userData` path via `app.setPath('userData', somePath)` although you'd only ever do this at the earliest possible point when the app starts and before it actually writes any Chromium data there.

It's common to change `userData` on Windows because it defaults in `~/AppData/Roaming/{{appName}}` and if this gets large, it causes issues with "roaming profiles" between machines in corporate environments. 

We've stored `sentry/scope_v2.json` in `userData` since forever and haven't had any complaints or issues.